knowledge-base/erlef/setup-beam/action-security.yml

```diff
@@ -0,0 +1,14 @@
+name: Setup Erlang/OTP with optional Elixir (and mix) and/or rebar3
+outbound-endpoints: 
+  - fqdn: repo.hex.pm
+    port: 443
+    reason: to download elixir and OTP builds
+  
+  - fqdn: github.com
+    port: 443
+    reason: to download gleam-lang & rebar3 releases 
+
+  - fqdn: api.github.com
```

Comment:
@h0x0er since this calls api.github.com, does it not use GitHub token?

Reply:
@varunsh-coder It is actually making an API request to public endpoint [click_here](https://github.com/erlef/setup-beam/blob/main/src/setup-beam.js#L249), so it doesn't requires GITHUB_TOKEN.
Thanks for reviewing..